portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSelectFromListField.testcase

```diff
@@ -72,18 +70,55 @@ definition {
 		property portal.acceptance = "false";
 
 		// TODO LPS-131358 CanBeEdited pending implementation
-
 	}
 
 	@description = "Verify that a Select From List Field can be required"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeRequired {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-131360 CanBeRequired pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Select from List");
+
+		FormFields.editOptionsRow(
+			optionName = "Option1",
+			optionValue = "Option 1");
+
+		Check.checkToggleSwitch(
+			key_toggleSwitchLabel = "Required Field",
+			locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		Button.clickSubmitButton();
+
+		FormPortlet.submitWithValidationError(validationErrorMessage = "This field is required.");
```

Comment:
@lucasperj the `submitWithValidationError` already clicks the submit button using the `FormPortlet.submit();` macro. I believe you don't need the `Button.clickSubmitButton();`. Can you please check? Thanks!